src/gui/widgets/LcdSpinBox.cpp

```diff
@@ -121,16 +120,16 @@ void LcdSpinBox::mouseMoveEvent( QMouseEvent* event )
 {
 	if( m_mouseMoving )
 	{
-		int dy = event->globalY() - m_origMousePos.y();
+		int dy = 3 * (event->globalY() - m_lastMousePos.y());
 		if( event->modifiers() & Qt::ShiftModifier )
 			dy = qBound( -4, dy/4, 4 );
 		if( dy > 1 || dy < -1 )
 		{
 			model()->setInitValue( model()->value() -
 						dy / 2 * model()->step<int>() );
```

Comment:
I think yo may multiply the factor here instead of changing `dy`.

Reply:
I saw that, too, but then, the ratio between non-shift- and shift-movement would have changed. The way it is now, the `dy` values should be close to those of `master`, and so the `qBound( -4, dy/4, 4 )` should have the same effect.